packages/trpc/server/routers/viewer/organizations/update.schema.ts

```diff
@@ -22,6 +25,7 @@ export const ZUpdateInputSchema = z.object({
   timeZone: z.string().optional(),
   weekStart: z.string().optional(),
   timeFormat: z.number().optional(),
+  metadata: teamMetadataSchema.optional(),
```

Comment:
Should we allow every single key to be update-able from the update mutation? One thing is to have a schema for reading the metadata. But IMO we should have another schema so not all keys are editable from here.